test/stores/oidc/OidcClientStore-test.ts

```diff
@@ -245,4 +236,17 @@ describe("OidcClientStore", () => {
             expect(OidcClient.prototype.revokeToken).toHaveBeenCalledWith(accessToken, "access_token");
         });
     });
+
+    describe("OIDC Aware", () => {
```

Comment:
I don't really know what "OIDC Aware" means in the context of a test for OidcClientStore. Surely `OidcClientStore` is always aware of OIDC?